modules/planning/scenarios/park_and_go/stage_adjust.cc

```diff
@@ -48,10 +48,22 @@ Stage::StageStatus ParkAndGoStageAdjust::Process(
     AERROR << "ParkAndGoStageAdjust planning error";
     return StageStatus::ERROR;
   }
-  const bool ready_to_cruise =
+  bool is_ready_to_cruise =
       scenario::util::CheckADCReadyToCruise(frame, scenario_config_);
 
-  if (!ready_to_cruise) {
+  bool is_end_of_trajectory = false;
+  const auto& history_frame = FrameHistory::Instance()->Latest();
+  if (history_frame) {
+    const auto& trajectory_points =
+        history_frame->current_frame_planned_trajectory().trajectory_point();
+    const auto trajectory_point_size =
+        history_frame->current_frame_planned_trajectory()
+            .trajectory_point_size();
+    auto last_point = trajectory_points[trajectory_point_size - 1];
```

Comment:
1.  `const auto&`
2. Simply use `trajectory_points.rbegin()->relative_time()`

Reply:
Fixed. Thank you!